lib/consumer/amqp_data_consumer.ex

```diff
@@ -15,7 +15,7 @@ defmodule Mississippi.Consumer.AMQPDataConsumer do
 
   # TODO should this be customizable?
   @reconnect_interval 1_000
-  @connection unless Mix.env() == :test, do: ExRabbitPoolConnection, else: MockAMQPConnection
+  @connection if Mix.env() != :test, do: ExRabbitPoolConnection, else: MockAMQPConnection
```

Comment:
You could evaluate avoiding negations whenever possible, in order to keep the code more readable:

```elixir
@connection if Mix.env() == :test, do: MockAMQPConnection, else: ExRabbitPoolConnection
```

Reply:
Nevermind, I saw that Styler does exactly that in the next commit.

Reply:
I actually disagree with Styler in that specific instance, but you can't individually opt-out, so it wins